### PR TITLE
Feature/new format on phone

### PR DESCRIPTION
**Pull request by leidyzulu**
New format phone
(XXX)XXX-XXXX

edit_text_phone

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PhoneNumberTextWatcherMask.kt [hunk 1]
```diff
@@ -19,51 +19,55 @@ package co.condorlabs.customcomponents.helper.masks
...
-
-    override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
-
-        s?.let {
-            if ((it.length == PHONE_NUMBER_FORMAT_FIRST_HYPHEN_INDEX ||
-                        it.length == PHONE_NUMBER_FORMAT_SECOND_HYPHEN_INDEX)
-            ) {
-                mReceiver.append(HYPHEN)
+            receiver.addTextChangedListener(this)
+            if (hadParenthesis and (mask == 1)) {
```
Comment (feguerrero) [inline app/src/main/java/co/condorlabs/customcomponents/helper/masks/PhoneNumberTextWatcherMask.kt]: Why `mask` is number type? If it is only used when is equals to 1, it should be a boolean.

File: test/src/main/res/layout/activity_mock.xml [hunk 2]
```diff
@@ -19,4 +19,5 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
+
```
Comment (feguerrero) [inline test/src/main/res/layout/activity_mock.xml]: Remove blank line.

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PhoneNumberTextWatcherMask.kt (hunk 1 above)
Comment (leidyzulu) [reply app/src/main/java/co/condorlabs/customcomponents/helper/masks/PhoneNumberTextWatcherMask.kt]: done

File: test/src/main/res/layout/activity_mock.xml (hunk 2 above)
Comment (leidyzulu) [reply test/src/main/res/layout/activity_mock.xml]: done

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/readme.md [hunk 3]
```diff
@@ -50,3 +50,23 @@ Emerald provides this component to be used as EditText
...
+# Custom EditTextPhoneField
+## Basic Usage
+
+```xml
+    <co.condorlabs.customcomponents.customedittext.EditTextPhoneField
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+```
+
+## Attributes
```
[the remark this answers is not in the document]
Comment (leidyzulu) [reply app/src/main/java/co/condorlabs/customcomponents/customedittext/readme.md]: done

**Pull request merged**